src/algebra/group/type_tags.lean

```diff
@@ -13,14 +13,54 @@ variables {α : Type u} {β : Type v}
 def additive (α : Type*) := α
 def multiplicative (α : Type*) := α
```

Comment:
Does `@[to_additive additive]` work here?

Reply:
Why would we want it?

Reply:
It seems less duplication to only define it once and have `to_additive` copy it, but maybe I'm overestimating the savings here.

Reply:
I mean, do we want `to_additive` to turn `multiplicative` in a lemma into `additive`? I don't think so. More precisely, I don't think that `to_additive` will work with a lemma involving both `α` and `multiplicative β`.